pkg/scheduler/metrics/metrics.go

```diff
@@ -95,16 +86,6 @@ var (
 			Help:           "Number of pending pods, by the queue type. 'active' means number of pods in activeQ; 'backoff' means number of pods in backoffQ; 'unschedulable' means number of pods in unschedulablePods that the scheduler attempted to schedule and failed; 'gated' is the number of unschedulable pods that the scheduler never attempted to schedule because they are gated.",
 			StabilityLevel: metrics.STABLE,
 		}, []string{"queue"})
-	// SchedulerGoroutines isn't called in some parts where goroutines start.
-	// Goroutines metric replaces SchedulerGoroutines metric. Goroutine metric tracks all goroutines.
-	SchedulerGoroutines = metrics.NewGaugeVec(
-		&metrics.GaugeOpts{
-			Subsystem:         SchedulerSubsystem,
-			DeprecatedVersion: "1.26.0",
```

Comment:
I would keep this a couple of more versions (so remove in 1.29), just because this metric was quite old.

Reply:
I followed your call on that one and reverted the change.

Just as an fyi, in 1.27 this metric will be hidden by default meaning that it won't be exposed to end-users unless they specifically tell the scheduler to expose it.

Reply:
oh is that a new change?